prog/vm/vm_pool.rb

```diff
@@ -22,14 +22,6 @@ def self.assemble(size:, vm_size:, boot_image:, location:, storage_size_gib:,
     end
   end
 
-  def before_run
-    when_destroy_set? do
-      unless ["destroy", "wait_vms_destroy"].include?(strand.label)
```

Comment:
Is this change safe?  `destroy` does not call `decr_destroy` in this prog, unlike other progs.

Reply:
I think it's safe. It's probably that `decr_destroy` was forgotten